ansible/group_vars/all/vars.yml

```diff
@@ -87,8 +87,8 @@ kiwix_data_directory: /mnt/content/wiki
 kiwix_directory_mode: "0755"
 
 kiwix_version: kiwix-tools_linux-armhf-3.4.0
-vikidia_version: vikidia_en_all_nopic_2023-03
-wiktionary_version: wiktionary_en_simple_all_nopic_2023-07
+vikidia_version: "vikidia_en_all_nopic_{{ lookup('pipe','date -d \"last month\" +%Y-%m') }}"
```

Comment:
Does this mean there is always a wiktionary version available with current date timestamp(year-month)?

Reply:
Yes. The site owner ensures that the wiktionary version of the previous month is available.